.github/actions/repo-gardening/src/get-next-valid-milestone.js

```diff
@@ -29,7 +29,7 @@ async function getNextValidMilestone( octokit, owner, repo, plugin = 'jetpack' )
 	for await ( const response of responses ) {
 		// Find a milestone which name is a version number
 		// and it's due dates is earliest in a future
-		const reg = new RegExp( plugin + '/d.d' );
+		const reg = new RegExp( plugin + '\\/\\d\\.\\d' );
```

Comment:
Could you explain what this does a bit? I'm not sure I understand the change.

Reply:
The original regexp string (`plugin/d.d`) would not match strings like "plugin/9.9". To make it work we need to do the following:
- escape `/`, since it is a special character in regexp
- to match any digit we should use `\d`. Simple `d` will match only latter "d"
- to match "." we should escape it: `\.` since `.` is a special character for regexp

after doing all the above, we'll end up with `plugin\/\d\.\d`. But since `\` is an escaping character in JS strings, we should escape these as well, thus double `\\`.

Reply:
`/` isn't a special character in regexp. It's the delimiter for regexp literals, so you need to escape it in something like `/foo\/bar/`, but you don't need to for `new RegExp( 'string' )` where delimiters aren't being used.

Not unless some environment is buggy, anyway.